jobs.html

```diff
@@ -9,11 +9,11 @@
             <h1>Compiler is a woman-owned software consultancy that’s passionate about making government tech solutions
                 accessible for all.</h1>
 
-            <h2 class="mt-5 fs-6">Open roles</h2>
-            <ul id="open" class="list-unstyled text-decoration-none"></ul>
+            <h2 class="mt-5 fs-6 pb-1 mb-2">Open roles</h2>
+            <ul id="open" class="list-unstyled d-flex flex-column gap-2 text-decoration-none"></ul>
 
-            <h2 class="mt-5 fs-6">Past roles</h2>
-            <ul id="closed" class="list-unstyled"></ul>
+            <h2 class="mt-5 fs-6 pb-1 mb-2">Past roles</h2>
+            <ul id="closed" class="list-unstyled d-flex flex-column gap-2"></ul>
```

Comment:
Use `d-flex flex-column gap-2` to create a 8px vertical gap (as in, padding) between the list items. `d-flex flex-column` sets the links to flow vertically, rather than horizontally and tells `gap` to create the gap vertically.